coccinelle/glnx-errors.cocci

```diff
@@ -0,0 +1,6 @@
+@@
+expression p;
+@@
+- glnx_set_error_from_errno (p);
+- goto out;
++ return  glnx_throw_errno (p);
```

Comment:
I haven't yet figured out enough Coccinelle to know if we can really do the "style rewrite" as a spatch, but this is a start.